Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Bot/trading-bot.json

```diff
@@ -125,7 +125,7 @@
                 },
                 {
                     "language": "DE",
-                    "text": "Trading Mines erleichtern die Entwicklung von Trading Bots, indem sie einen Rahmen für die Erstellung unterteilter Definitionen für die Prozesse und Produkte der Bots bieten, ähnlich wie Data Mines es für Sensoren und Indikatoren tun.",
+                    "text": "Trading Mines erleichtern die Entwicklung von Trading Bots, indem sie einen Rahmen für die Erstellung unterteilter Definitionen für die Prozesse und Produkte der Bots bieten, ��hnlich wie Data Mines es für Sensoren und Indikatoren tun.",
```

Comment:
The ä doesn't seem to be detected properly, you can try to fix it and do a commit again or if that's too cumbersome for you I can merge it and afterwards we'll adjust it. :+1:

Reply:
Hi, is the translation to German Automatic ? I didn t write it

Reply:
Done